zebra-network/src/peer/connection.rs

```diff
@@ -92,6 +92,25 @@ impl fmt::Display for Handler {
 }
 
 impl Handler {
+    /// Returns the Zebra internal handler type as a string.
+    pub fn command(&self) -> String {
+        match self {
+            Handler::Finished(Ok(response)) => format!("Finished({})", response.command()),
+            Handler::Finished(Err(error)) => format!("Finished({})", error),
+
+            Handler::Ping(_) => "Ping".to_string(),
+            Handler::Peers => "Peers".to_string(),
+
+            Handler::FindBlocks { .. } => "FindBlocks".to_string(),
+            Handler::FindHeaders { .. } => "FindHeaders".to_string(),
+
+            Handler::BlocksByHash { .. } => "BlocksByHash".to_string(),
+            Handler::TransactionsById { .. } => "TransactionsById".to_string(),
+
+            Handler::MempoolTransactionIds => "MempoolTransactionIds".to_string(),
+        }
+    }
```

Comment:
Idea: not sure if this mini-optimization is useful, but it would avoid a heap allocation if it's not needed:

```suggestion
    pub fn command(&self) -> Cow<'static, str> {
        match self {
            Handler::Finished(Ok(response)) => format!("Finished({})", response.command()).into(),
            Handler::Finished(Err(error)) => format!("Finished({})", error).into(),

            Handler::Ping(_) => "Ping".into(),
            Handler::Peers => "Peers".into(),

            Handler::FindBlocks { .. } => "FindBlocks".into(),
            Handler::FindHeaders { .. } => "FindHeaders".into(),

            Handler::BlocksByHash { .. } => "BlocksByHash".into(),
            Handler::TransactionsById { .. } => "TransactionsById".into(),

            Handler::MempoolTransactionIds => "MempoolTransactionIds".into(),
        }
    }
```

Reply:
Fixed in 6f62fe2 in PR #3203.